packages/integration-tests/src/integration/disk-encryption/step-0-disk-encryption.spec.js

```diff
@@ -0,0 +1,92 @@
+import { clusterDetailsPage } from '../../views/clusterDetails';
+import { diskEncryptionPage } from '../../views/diskEncryption';
+import { bareMetalDiscoveryPage } from '../../views/bareMetalDiscovery';
+import { commonActions } from '../../views/common';
+import {diskEncryptionValues, tangServerValues} from '../../fixtures/disk-encyrpiton'
+import * as utils from '../../support/utils';
+import {transformBasedOnUIVersion} from "../../support/transformations";
+
+const fillTangServers = (index) => {
+  const tangServerUrl = diskEncryptionPage.getTangServerUrl(index);
+  tangServerUrl.type(tangServerValues.Url);
+  tangServerUrl.should('have.value', tangServerValues.Url);
+  const tangServerThumbprint = diskEncryptionPage.getTangServerThumbprint(index);
+  tangServerThumbprint.type(tangServerValues.Thumbprint);
+  tangServerThumbprint.should('have.value', tangServerValues.Thumbprint);
+};
+
+const enableDiskEncryption = () => {
+  diskEncryptionPage.getWorkersEncryptionSwitch().click({force: true});
+  diskEncryptionPage.getEncryptionMode().first().should('be.checked');
+  diskEncryptionPage.getEncryptionMode().first().should('have.value', 'tpmv2');
+  diskEncryptionPage.getWorkersEncryptionSwitch().click({force: true});
+};
+
+const enableTangServers = () => {
+  diskEncryptionPage.getEncryptionMode().check("tang");
+  diskEncryptionPage.getTangServerUrl(0).should('be.visible');
+  diskEncryptionPage.getTangServerThumbprint(0).should('be.visible');
+  diskEncryptionPage.getEncryptionMode().first().check();
+};
```

Comment:
It looks like the `diskEncryptionPage` is the sole entity responsible for all these 3 actions. Wouldn't it be better to let it expose them as methods? I believe it will be easier to find them if we need to reuse this logic.

Reply:
I'm taking back this functionality is not being reused.
I would just interpolate the code inside this test case:
```javascript
it('Can use disk encryption', () => {
      enableDiskEncryption();
      enableTangServers();
});
```